nekoyume/Assets/_Scripts/UI/Module/GrindModule.cs

```diff
@@ -298,7 +314,7 @@ private static bool CheckSelectedItemsAreStrong(List<Equipment> equipments)
                 item.level > 0 || item.Skills.Any() || item.BuffSkills.Any());
         }
 
-        private void Action(List<Equipment> equipments)
+        private void Action(List<Equipment> equipments, bool chargeAp)
```

Comment:
Is `chargeAp` parameter needed? Action point is already checked in `CheckApPotionForAction`.